### PR TITLE
Merged two PRs and added make_html.py

### PR DESCRIPTION
**Pull request by rjleveque**
I merged the PRs #68 and #69 into this one since they were starting to diverge and I wanted to test various changes from each of them together.  I suggest we close both of those if @ketch and @maojrs are happy with this one, and then merge this in and move on from there with new PRs.

I added a `make_html.py` to create html files from all the notebooks and put them in a separate directory `riemann_book_files` that can be moved to a website, such as: http://faculty.washington.edu/rjl/riemann_book/Index.html

This is far from perfect since much of the rendering is not working properly.  See for example [the Tammann](http://faculty.washington.edu/rjl/riemann_book/Euler_equations_TammannEOS.html) notebook, which shows many of the problems:

 - bib citations mess up the rest of the text beyond
 - equation numbers aren't showing up
 - javascript widget broken.

On the other hand the [Introduction](http://faculty.washington.edu/rjl/riemann_book/Introduction.html) looks pretty good except for the equation numbers.

### CODE REVIEW

**Comment by ketch** [comment]
Could we merge #69 and then review this for merging?

**Comment by ketch** [comment]
I notice that the stripe plots in the HTML use rasterized images.  What's the rationale for going that route?

**Comment by rjleveque** [comment]
I merged #69 and then did a test merge of this branch and it seems to merge without any conflicts.  But @ketch should try it out and then merge if all seems ok.

Github seems to be flagging mostly a bunch of conflicting metadata changes that merge fine, but maybe because I have nbdime installed?  We might have to merge locally and push back to Github rather than using the Merge button?

**Comment by rjleveque** [comment]
@ketch: No good reason for using rasterized images but I'm not sure where
that's being done.  If you know how to fix it, please go ahead.

**Comment by rjleveque** [comment]
@ketch: I tried changing from `png` to `svg` in the function `utils.animation_tools.make_image` but it seems `imread` only supports `png` format, see http://stackoverflow.com/questions/31452451/importing-an-svg-file-a-matplotlib-figure

 So it doesn't seem easy to make JSAnimations out of vector graphics files, but if you know a way please let me know.

Otherwise I will go ahead and merge this in the next day or two so that we can move forward with submitting the proposal.

**Comment by ketch** [comment]
To make JSAnimations without rasterizing, I use the images from memory rather than writing them to disk.  I did this for instance in the Shallow_tracer notebook; here's an executed version with the animation working:

http://nbviewer.jupyter.org/gist/ketch/aa3c953fb06d95cfb2624b5110c0a7ff

**Comment by ketch** [comment]
I have a vector-image based HTML-functional animation working and will push something soon (to #73).

File: Euler_equations_TammannEOS.ipynb [hunk 1]
```diff
@@ -340,7 +335,23 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "collapsed": false
+    "collapsed": false,
+    "nbdime-conflicts": {
```
Comment (ketch) [inline Euler_equations_TammannEOS.ipynb]: It looks like some nbdime merge conflict stuff got included in this commit.

**Comment by ketch** [comment]
Okay, I closed #73 and I'm just looking over this PR (#71).  It seems indeed that somewhere there was a merged that messed things up.  For instance, if you look at the first Traffic notebook:

https://github.com/rjleveque/riemann_book/blob/4368d3a129765e799d376277d80d9ee195eb9b32/Traffic_flow.ipynb

you'll see that the velocity figure is duplicated, all the imports are duplicated, and a bunch of changes I made have been reverted.

It's going to be very difficult to build on this and try to make sure that we didn't lose any of our previous work.  I think that the best path forward is to abandon this PR and put the relevant changes from it into one or more new PRs based off the current master.

I'm sorry for being a bit unresponsive and slow to review/merge PRs.  I can definitely be more proactive about this now.

**Comment by rjleveque** [comment]
Sorry, I must have messed things up when merging `bookform` and `plot_red_blue_stripes`, so maybe we can try a new merge of those and be more careful about it.

**Comment by rjleveque** [comment]
This was replaced by #74.

**Pull request closed without merging**